src/app/views/account/account.component.html

```diff
@@ -89,7 +89,7 @@ <h2 class="col-9">myTOA</h2>
           <div mdcSubtitle1 class="mt-1">{{ 'pages.account.api_card.your_key' | translate }}</div>
           <code dir="ltr">{{ user.apiKey || 'Not found' }}</code>
           <div mdcSubtitle1 class="mt-2">{{ 'pages.account.api_card.docs' | translate }}
-            <a href="https://orange-alliance.github.io/TOA-API" target="_blank" (click)="this.sendAnalytic('leaving', '/account', 'api docs');">{{ 'general.here' | translate }}</a>.
+            <a routerLink="apidocs" target="_blank" (click)="this.sendAnalytic('leaving', '/account', 'api docs');">{{ 'general.here' | translate }}</a>.
```

Comment:
Great. Didn't notice this until now, but since this link is now internal, rather than an external site, we can remove the analytic by dropping the `(click)="this.sendAnalytic('leaving', '/account', 'api docs');"`

Reply:
Done!